web/html/src/manager/minion/ansible/edit-ansible-path.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import { TextField } from "components/fields";
+import { AsyncButton, Button } from "components/buttons";
+
+const EditAnsiblePath = (props) => {
+  return (
+    <>
+      <div className="d-block">
+        <div className="col-md-10">
+          <TextField
+            value={props.ansiblePath.path}
+            onChange={(e: any) => props.editPath(e.target.value.toString())}
```

Comment:
Out of curiosity, is the type of `e` not automatically inferred here? You can make a ticket for me to look into this if that's the case, `onChange` should automatically be typed as `React.ChangeEvent<HTMLInputElement>` or something similar imo.

Reply:
I pushed a change that should fix this, the prop declaration in `TextField` was slightly wrong.